configs/aarch64_classic_defconfig

```diff
@@ -89,6 +89,7 @@ BR2_PACKAGE_WATCHDOGD=y
 BR2_PACKAGE_MG=y
 BR2_PACKAGE_MOST=y
 BR2_PACKAGE_NANO=y
+BR2_PACKAGE_FRR=y
```

Comment:
Not needed in Classic build.  Classic is for non-netconf old-style systems where you manually edit a persistent `/etc` :-)